tests/v2/validation/provisioning/rke2/custom_cluster.go

```diff
@@ -26,7 +26,7 @@ const (
 	namespace = "fleet-default"
 )
 
-func TestProvisioningRKE2CustomCluster(t *testing.T, client *rancher.Client, externalNodeProvider provisioning.ExternalNodeProvider, nodesAndRoles []string, kubeVersion, cni string, harden *provisioning.Config, nodeCountWin int, hasWindows bool) {
+func TestProvisioningRKE2CustomCluster(t *testing.T, client *rancher.Client, externalNodeProvider provisioning.ExternalNodeProvider, nodesAndRoles []string, kubeVersion, cni string, harden *provisioning.Config, nodeCountWin int, hasWindows bool, config *provisioning.Config) {
```

Comment:
```suggestion
func TestProvisioningRKE2CustomCluster(t *testing.T, client *rancher.Client, externalNodeProvider provisioning.ExternalNodeProvider, nodesAndRoles []string, kubeVersion, cni string, config *provisioning.Config, nodeCountWin int, hasWindows bool) {
```
Looks like we are passing in the same variable twice under a different name

Reply:
Yeah, the `harden` is expected since we need that for QA's hardening tests we run. I'm going to refactor a bit here to not pass the whole config for PSACT.